README.md

```diff
@@ -176,3 +176,15 @@ or add the option
   -DENABLE_CODECOV=ON
   ```
 if compiling with CMake.
+
+### Build the manual locally
+
+The documentation can be built locally using `sphinx`.
+Additionally, `sphinx_rtd_theme` and `nbsphinx` modules are required.
+  ```
+  cd <homedir>/src/forte/docs
```

Comment:
```suggestion
from the forte source directory (e.g., $HOME/src/forte) change to the directory `docs` and call `make html`:
  cd docs
```

Reply:
I remove "$HOME/" from the suggestion because it is not the way we describe the directory at the beginning of this README.

I wonder if I should move this part inside `docs`. Maybe that fits better.

Reply:
I would move it to the docs. Note that if you want to modify docs in notebooks, you will have to change the corresponding ipynb first, then call a script in the docs folder (all of this is documented).

Reply:
Moved.